src/main/resources/i10n/SuccessFactorsBatchSourceBundle.properties

```diff
@@ -30,6 +31,8 @@ err.missing.dataservice.version=No data service version number found. Please ens
 
 ## SAP SuccessFactors - metadata processing related messages
 err.no.column.found=''{0}'' not found in the ''{1}'' entity.
+err.unsupported.associatedEntity=''{0}'' not found in the ''{1}'' entity. Please ensure that associated entity is correct. Please check the property name in Admin Center > OData API Data Dictionary or entity metadata.
+
```

Comment:
message can be refactored as
`'{0}'' not found in the ''{1}'' entity. Please ensure that the associated entity is correct or check the property name in Admin Center > OData API Data Dictionary or entity metadata.`

Reply:
done.